Client/Client.Qt/Pages/ChannelListPage.cpp

```diff
@@ -48,25 +48,15 @@ void ChannelListPage::addChannelToMainChannelWidget()
 {
     if (_channelList->currentItem())
     {
-        if (_widgetChannelList->count() != 0)
+        std::string channel = _channelList->currentItem()->text().toStdString();
+        auto        channelInfo =
+            std::find_if(channels.begin(), channels.end(), [&channel](const Network::ChannelInfo& i) 
```

Comment:
I guess, if you don't change the var in lambda that you should pass it by value ([&channel] -> [channel]).